front/app/modules/commercial/widgets/index.tsx

```diff
@@ -1,22 +1,21 @@
 import React from 'react';
 import { ModuleConfiguration } from 'utils/moduleUtils';
-const Tab = React.lazy(() => import('./admin/components/Tab'));
 
 const AdminWidgetsContainerComponent = React.lazy(
   () => import('./admin/containers')
 );
 
 const configuration: ModuleConfiguration = {
   routes: {
-    'admin.settings': [
+    'admin.tools': [
       {
         path: 'widgets',
         element: <AdminWidgetsContainerComponent />,
       },
     ],
   },
   outlets: {
-    'app.containers.Admin.settings.tabs': (props) => <Tab {...props} />,
+    'app.containers.admin.tools': React.Fragment,
   },
```

Comment:
I'm not sure if I understand this. Why are we adding this outlet? It looks like it's not being used, besides rendering an empty Fragment.

Reply:
Good point!